packages/service-client/odsp-client/README.md

```diff
@@ -0,0 +1,101 @@
+## @fluid-experimental/odsp-client
+
+The odsp-client package provides a simple and powerful way to consume collaborative Fluid data with the ODSP service.
+
+## Using odsp-client
+
+The odsp-client package has a OdspClient class that allows you to interact with Fluid
+
+```typescript
+import { OdspClient } from "@fluid-experimental/odsp-client";
+```
+
+### Example usage
+
+```typescript
+import { OdspClient, OdspConnectionConfig, OdspClientProps } from "@fluid-experimental/odsp-client";
+
+const connectionConfig: OdspConnectionConfig = {
+	tokenProvider: "<YOUR_TOKEN_PROVIDER>",
+	siteUrl: "<SITE_URL>",
+	driveId: "<RAAS_DRIVE_ID>",
+	path: "<FOLDER_PATH>",
+};
+
+export const clientProps: OdspClientProps = {
+	connection: connectionConfig,
+};
+
+const client = new OdspClient(clientProps);
+```
+
+### Experimental Features
+
+`OdspClient` supports the ability to instantiate with experimental features enabled.
+These features are experimental in nature and should **NOT** be used in production applications.
+To learn more, see [Experimental Features](https://fluidframework.com/docs/build/experimental-features/).
+
+## Fluid Containers
+
+A Container instance is a organizational unit within Fluid. Each Container instance has a connection to the defined Fluid Service and contains a collection of collaborative objects.
```

Comment:
I do not like that articulation. Not clear what is "connection to defined Fluid Service" (does it mean AFR or ODSP?)
Container maps directly to a file - a unit of storage, with all of its semantics (permissions), but also Fluid adds its own semantics (ordering of ops is per container).